crates/ironrdp-connector/Cargo.toml

```diff
@@ -22,11 +22,15 @@ arbitrary = ["dep:arbitrary"]
 ironrdp-svc = { path = "../ironrdp-svc", version = "0.4" } # public
 ironrdp-core = { path = "../ironrdp-core", version = "0.1" } # public
 ironrdp-error = { path = "../ironrdp-error", version = "0.1" } # public
-ironrdp-pdu = { path = "../ironrdp-pdu", version = "0.5", features = ["std"] } # public
+ironrdp-pdu = { path = "../ironrdp-pdu", version = "0.5", features = [
+    "std",
+] } # public
 arbitrary = { version = "1", features = ["derive"], optional = true } # public
 sspi = "0.15" # public
 url = "2.5" # public
-rand_core = { version = "0.6", features = ["std"] } # TODO: dependency injection?
+rand_core = { version = "0.6", features = [
+    "std",
+] } # TODO: dependency injection?
```

Comment:
nitpick: Revert this change since it’s unrelated with the current PR, and this is also not particularly a common style among the Rust community